docs/mkdocs/secureboot-build-and-sign.md

```diff
@@ -0,0 +1,86 @@
+# Signing kernel modules using KMM
+
+### Secure-boot with KMM
+
+For more details on using secure-boot see [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_monitoring_and_updating_the_kernel/signing-kernel-modules-for-secure-boot_managing-monitoring-and-updating-the-kernel) or [here](https://wiki.debian.org/SecureBoot)
+
+### Using Signing with KMM
+
+On a secure-boot enabled system all kernel modules (kmods) must be signed with a public/private keypair enrolled into the Machine Owner's Key or MOK database. Drivers distributed as part of a distribution should already be signed by the distributions private key, but for kernel modules build out-of-tree KMM supports signing kernel modules using the "sign" section of the kernel mapping.
+
+KMM has the ability to build and sign kernel modules together by defining both the `build` and `sign` sections in the Module custom resource
+
+To use this functionality you need:
+
+ * A public private key pair in the correct (der) format
+ * At least one secure-boot enabled nodes with the public key enrolled in its MOK database
+ * The source code for the kmod to build and a Dockerfile for the driver container that will include it
+
+The yaml below should build a new container image using the source code from the https://github.com/chr15p/simple_driver_container (this kernel module does nothing useful but provides a good example). The image produced is saved back the the registry with a temporary name, and this temporary image is then signed using the parameters in the `sign` section.
```

Comment:
Any reason we are not using https://github.com/kubernetes-sigs/kernel-module-management/tree/main/ci/kmm-kmod instead of https://github.com/chr15p/simple_driver_container?

This is yet another derivative of https://github.com/ybettan/simple-kmod. I don't think we should have that many different examples.

Reply:
just because that's the version I was testing with and I wanted to make sure I had an example that actually worked!  I think we should try and make examples as consistent as possible across all the docs through so if  https://github.com/kubernetes-sigs/kernel-module-management/tree/main/ci/kmm-kmod is the definitive version thats great, I will update

Reply:
Thanks. https://github.com/kubernetes-sigs/kernel-module-management/tree/main/ci/kmm-kmod is the one we are using in CI and referring to in our docs https://github.com/kubernetes-sigs/kernel-module-management/blob/65421fee9d35844250160398febdc102fa00b0c3/docs/labs/ubuntu/kmm-kmod.yaml#L40-L41